<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Profo_info</class>
 <widget class="QDialog" name="Profo_info">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>886</width>
    <height>707</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Profo infomation</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(40, 40, 40);
color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>866</width>
        <height>687</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="tach_vale_label">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Positions</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="Positions_tableView"/>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Transactions</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="Transactions_tableView"/>
       </item>
       <item>
        <widget class="QPushButton" name="profo_csv">
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(25, 69, 85);</string>
         </property>
         <property name="text">
          <string>Download CSV</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="prof_close">
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(25, 69, 85);</string>
         </property>
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>prof_close</sender>
   <signal>clicked()</signal>
   <receiver>Profo_info</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>405</x>
     <y>675</y>
    </hint>
    <hint type="destinationlabel">
     <x>516</x>
     <y>704</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
